### ISSUE TITLE
error calling testsComplete when testing package

### ISSUE THREAD

**Issue by marcuscarr** [mad-eye/meteor-mocha-web#194]
Moving this from meteor-velocity/velocity#398

When I try to test a Meteor package I am writing with `mike:mocha-package`, the following error shows on the server:
```
error calling testsComplete function { [Error: Match error: Expected string, got undefined]
  message: 'Match error: Expected string, got undefined',
  path: '',
  sanitizedError:
   { [Error: Match failed [400]]
     error: 400,
     reason: 'Match failed',
     details: undefined,
     message: 'Match failed [400]',
     errorType: 'Meteor.Error' },
  errorType: 'Match.Error' }
```

The same error is seen even with a just-created meteor package (`meteor create --package`). The test command I am using is `meteor test-packages --driver-package velocity:html-reporter ./`, and you can reproduce the error with [this package](https://github.com/marcuscarr/dummy-package).

If I use `meteor test-packages --driver-package velocity:console-reporter ./` I get the same error.

I tried adding version constraints for all the packages, and still got the same error. Running the one of the commands suggested in mad-eye/meteor-mocha-web#184 (`meteor test-packages --velocity ./`) gave the same error. The other command (`meteor test-packages --velocity --driver-package respondly:test-reporter ./`) gives this error in the browser console: 
```
Uncaught TypeError: $ is not a function
    (anonymous function) @ respondly_util.js:5911
    runStartupCallbacks @ meteor.js:759
    ready @ meteor.js:761
```

Using `--release velocity:METEOR@1.2.0.2_1` doesn't fix it either.

The tests themselves seem to run as expected (screenshot below), but random background errors make me nervous.

<img width="1376" alt="screen shot 2015-10-07 at 11 26 51" src="https://cloud.githubusercontent.com/assets/4015075/10346953/81361648-6ce6-11e5-8fba-fc7fb4a660cd.png">

**Comment by rissem** [comment]
Should be resolved with mike:mocha-package@0.5.9

Let me know if you're still seeing anything, and sorry this took me so long for me to get to.

**Comment by marcuscarr** [comment]
All fixed. Thanks!
